hpicker.js

```diff
@@ -262,9 +274,9 @@ class HorizontalPicker extends Component {
     const itemValue = child.props.value;
     const color = this.props.foregroundColor || defaultForegroundColor;
     return (
-      <TouchableWithoutFeedback key={itemValue} onPress={v => this.handleItemPress(v)}>
+      <TouchableWithoutFeedback key={itemValue} onPress={child.props.onPress}>
```

Comment:
Why bind directly the `onPress` to the item? There's a dedicated handler to trigger changes when an item is tapped, and this would defeat the purpose.

Reply:
the general `handleItemPress` does not recognize which item was pressed. This is why I had to have `onPress` on each child. I will see if I can change this by passing the item value instead of `v`

Reply:
@mkko I've just changed it to properly call handleItemPress. Just passed the item value